aspnetcore/fundamentals/logging/index/samples/3.x/LoggingConsoleApp/Program.cs

```diff
@@ -5,7 +5,7 @@ class Program
 {
     static void Main(string[] args)
     {
-        var loggerFactory = LoggerFactory.Create(builder =>
+        using var loggerFactory = LoggerFactory.Create(builder =>
```

Comment:
if the factory is not disposed, then logs are not displayed in console. So a user who simply copy pastes this snippet into a console app, will find it hard to understand why this example i not working. 
(Alternately to support older C# version, we can modify this to explicitly call loggerFactory.Dispose() at the end.

Either way - this PR makes this example runnable without any confusion around why I am not seeing anything in console.

Reply:
Happy to make this change. I don't think we need to complicate for 2.1 folks who may not be using C# 8